### DISCUSSION TITLE
Challenge contracts?

### DISCUSSION THREAD

**Discussion by codex-scribe** [Cyfrin/foundry-full-course-cu#3127] [Q&A]
![image](https://github.com/user-attachments/assets/d64110bf-a674-493a-981d-ee593d0b4d5d)

I just finished the Storage Factory lesson from Solidity 101.  At the end of this video, it says to try the challenge contracts.  I would love to do that since I always like to learn more, but I don't know where they are.

**Comment by EngrPips** [comment]
Hello @codex-scribe, You should read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018 and it will help you understand how to find the challenges and how to solve them.
